app/models/parameter.rb

```diff
@@ -34,8 +33,7 @@ def set_priority
     self.priority = PRIORITY[t.to_s.underscore.to_sym] unless t.blank?
   end
 
-  def strip_whitespaces
-    self.name = self.name.strip unless name.blank? # when name string comes from a hash key, it's frozen and cannot be modified
```

Comment:
this is old leftovers, the `StripWhitespace` concern already takes care of this.